doc-surrealdb_versioned_docs/version-1.x/surrealql/statements/update.mdx

```diff
@@ -29,13 +29,15 @@ UPDATE [ ONLY ] @targets
 ;
 ```
 
+`@target` refers to either record output including an `id` field, or even just a record ID on its own.
+
 ## Example usage
 
 The following query shows example usage of this statement.
 
 ```surql
 -- Update all records in a table
-UPDATE person SET skills += ['breathing'];
+UPDATE person SET skills += 'breathing';
```

Comment:
I wonder if we should leave this example as is because `SET` automatically creates an array.

Reply:
I guess either way has both good and bad points:

* Leave the brackets in: makes it clear that this is an array, but gives the impression that you need [] to make the query work
* Remove the brackets out: doesn't give the impression that [] is needed, but may not be clear to the reader that += alone will result in an array

Maybe a note above the line like this?
-- The `skills` field is an array<string>. The += operator alone is enough for SurrealDB to infer the type